stdlib/package.mill

```diff
@@ -0,0 +1,21 @@
+package build.stdlib
+
+import mill._
+import mill.scalalib._
+import mill.scalalib.scalafmt._
+import mill.define.Cross
+
+import build._
+
+object `package` extends RootModule {
+  // https://github.com/com-lihaoyi/mill/issues/3693
+  object cross extends Cross[Stdlib](v.scalaCrossVersions)
+}
+
+trait Stdlib extends CrossSbtModule with HasScala2Plugin with ScalafmtModule {
```

Comment:
Not in this PR, the Stdlib is introduced by me, but seems never published, I think we can remove it in the future.